sourcegraph/sourcegraph
# Code Insights: Add pings for the standalone insight page

Closes https://github.com/sourcegraph/sourcegraph/issues/36092

[Related RFC](https://docs.google.com/document/d/1wDZRxiWBVbLs5Xh13rarBOGZS2ZekppSOSFIdGlkp9Q/edit#)

## Test plan
- Make sure that you can see all pings for the standalone insight page from the RFC 
  - Edit clicks count (weekly and total)
  - Filters change count
  - Standalone insight page views count (weekly and total)
- Make sure that we capture all added pings in the code insight pings page
